src/researchhub_comment/related_models/rh_comment_model.py

```diff
@@ -136,7 +136,7 @@ def users_to_notify(self):
 
     def update_comment_content(self):
         celery_create_comment_content_src.apply_async(
-            (self.id, self.comment_content_json), delay=5
+            (self.id, self.comment_content_json), countdown=2
```

Comment:
can you explain what countdown does?

Reply:
The amount of time before Celery can execute the task